README.md

```diff
@@ -2,39 +2,49 @@
  
 # Table of Contents
 
-* [Allgemeines](#allgemeines)
-   * [Schaufensterkonditionen](#schaufensterkonditionen)
-* [Beispiele](#beispiele)
-   * [Query topSchaufensterkondition](#query-topschaufensterkondition)
-      * [POST Request](#post-request)
-      * [POST Response](#post-response)
-   * [Query schaufensterkonditionen](#query-schaufensterkonditionen)
-      * [POST Request](#post-request-1)
-      * [POST Response](#post-response-1)
-* [Request](#request)
-   * [Authentifizierung](#authentifizierung)
-   * [Nachverfolgbarkeit von Requests](#nachverfolgbarkeit-von-requests)
-   * [Format](#format)
-   * [Request Parameter](#request-parameter)
-      * [Query topSchaufensterkondition](#query-topschaufensterkondition-1)
-      * [Query schaufensterkonditionen](#query-schaufensterkonditionen-1)
-      * [Partner-ID](#partner-id)
-      * [Datenkontext](#datenkontext)
-      * [Finanzierungszweck](#finanzierungszweck)
-   * [Anfragbare Felder](#anfragbare-felder)
-      * [Schaufensterkondition](#schaufensterkondition)
-         * [Gesamtkonditionen](#gesamtkonditionen)
-            * [Konditionsspanne](#konditionsspanne)
-              * [Konditionsgrenze](#konditionsgrenze)
-         * [Ratenkredit](#ratenkredit)
-            * [Produktanbieter](#produktanbieter)
-            * [Anschrift](#anschrift)
-            * [Logo](#logo)
-* [Fehlercodes](#fehlercodes)
-   * [HTTP-Status Errors](#http-status-errors)
-   * [Weitere Fehler](#weitere-fehler)
-* [Tools](#tools)
-* [Nutzungsbedingungen](#nutzungsbedingungen)
+- [Allgemeines](#allgemeines)
+  * [Authentifizierung](#authentifizierung)
+  * [Nachverfolgbarkeit von Requests](#nachverfolgbarkeit-von-requests)
+  * [Request](#request)
+  * [Fehlercodes](#fehlercodes)
+    + [HTTP-Status Errors](#http-status-errors)
+    + [Weitere Fehler](#weitere-fehler)
+  * [Tools](#tools)
+- [Schaufensterkonditionen](#schaufensterkonditionen)
+  * [Query Top-Schaufensterkondition](#query-top-schaufensterkondition)
+    + [Request](#request-1)
```

Comment:
Mir ist gerade kein Kniff in Markdown bekannt, der das löst, daher nur als Gedanke: kann das Fragment `#request-1` anders gebaut werden, beispielsweise `#top-schaufensterkondition-request`? Das würde die Links stabiler machen.

Reply:
Evtl. klappt diese Syntax für custom header ids: https://www.markdownguide.org/extended-syntax/#heading-ids

Reply:
ich benutze da einfach ein Tool zum generieren, ist mir stabil genug ...

Reply:
Welches Tool? Ggf. verbreiten wir das als Tipp, damit bei manuellen Umsortierungen durch andere Leute die Links nicht kaputt gemacht werden oder gar an die falsche Stelle zeigen... ein Reviewer wird das möglicherweise auch nicht bemerken ;)

Reply:
ich benutze `markdown-toc`, das ist ein npm-Module. Das kann man direkt auf die Readme anwenden.